src/store/modules/ui.store.js

```diff
@@ -239,10 +239,11 @@ export default {
         setCompareRatio({ commit }, value) {
             /*
                 This check is here to make sure the compare ratio doesn't get out of hand
-                The logic is, if we try to set a value outside of the range at which the compare
-                slider would be visible, we set it to its default value instead
+                The logic is, we want the compare ratio to be either in its visible range,
+                which is 0.001 to 0.999, and it's "storage range" (-0.001 to -0.999). If
+                we are not within these bounds, we revert to the default value (-0.5)
             */
-            if (0.0 < value && value < 1.0) {
+            if (-1.0 < value && value < 1.0 && value != 0.0) {
                 commit('setCompareRatio', value)
             } else {
```

Comment:
if you follow what I suggested above, you might want to revert this to checking between 0 and 1